src/main/kotlin/org/jitsi/jibri/util/ProcessWrapper.kt

```diff
@@ -98,7 +99,13 @@ class ProcessWrapper(
         // because we want them to read everything available from the
         // process' inputstream. Once it's done, they'll read
         // the EOF and close things up correctly
-        runtime.exec("kill -s SIGINT ${process.pidValue}")
+
+        var output: OutputStream = process.outputStream
+
+        output.write("q\n".toByteArray())
+        output.flush()
+
+        runtime.exec("bash -c \"kill -s SIGINT ${process.pidValue}\"")
```

Comment:
I don't know why this line does not work for me, so I have to send a `q` command...

Reply:
Hmm, this does seem a bit weird and this isn't a problem I've seen with killing ffmpeg.  I think we'll need to understand this better before merging.  Is this unique to the docker environment?

Reply:
I am not sure about that. I don't have time to test this in a native environment.

`java.io.IOException: Cannot run program "kill": error=2, No such file or directory with stack: `
I found some issues about this error on StackOverflow and they suggest using `bash -c`

After I updated it with `bash -c`, the FFmpeg still does not stop gracefully. So, I try to send a `q` command and it works

Here is the information about my deployment:
- Host: AWS Ubuntu 20.04.1 LTS  5.4.0-56-generic
- Docker: 19.03.14
- Docker Compose: 1.27.4, build 40524192
- Pulled to the latest images (11-12-2020)

The java version of jibri container:
```
Debian GNU/Linux 10 (buster)

openjdk version "1.8.0_275"
OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_275-b01)
OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.275-b01, mixed mode)
```

Reply:
Hello, I applied this and recompiled. Successfully fixed corrupt video files and "ffmpeg not stop gracefully" for my docker-compose installation.